src/assets/translations/fr/main.json

```diff
@@ -557,14 +598,21 @@
     },
     "deposit": {
       "amountToDeposit": "Montant à déposer",
+      "depositInto": "Déposer dans %{opportunité}",
       "estimatedReturns": "Rendement annuel estimé*",
       "estimateDisclaimer": "*Les estimations fluctuent en fonction des conditions du marché.",
       "footerDisclaimer": "En continuant, vous acceptez nos",
       "footerDisclaimerLink": "conditions d'utilisation.",
-      "slippageSettings": "Paramètres de glissement de taux"
+      "slippageSettings": "Paramètres de glissement de taux",
+      "status": {
+        "pending": "Votre transaction a été diffusée et est en attente de confirmation.",
+        "success": "Vous avez effectué un dépôt avec succès dans %{opportunity}.",
+        "failed": "Il y a eu un problème avec votre dépôt."
+      }
     },
     "withdraw": {
       "amountToWithdraw": "Montant à retirer",
+      "withdrawFrom": "Retirer de %{opportunité}",
```

Comment:
Interpolation variables shouldn't be translated

Reply:
Addressed in https://github.com/shapeshift/web/pull/2220/commits/b21538aa8648db3efa253c57ceacc30df3f9871b